config.env.example

```diff
@@ -4,6 +4,9 @@ CRN_SERVER_URL=http://localhost:9876
 # Secret authentication value for this instance
 JWT_SECRET=
 
+# Coral Talk Secret from SSO Config in Admin Panel
+CORAL_JWT_SECRET=ssosec_0178013126e99c4f68e06d47794a96c01c5a117b01e00687f24c310a2b49d9f993
```

Comment:
This should be blank, it shouldn't be possible to accidentally start without setting a new random value.

Reply:
oof, cut and paste